mathics/builtin/arithfns/basic.py

```diff
@@ -571,7 +571,7 @@ class Power(BinaryOperator, _MPMathFunction):
             'Infix[{HoldForm[x], HoldForm[y]}, "^", 590, Right]'
         ),
         ("", "x_ ^ y_"): (
-            "PrecedenceForm[Superscript[OuterPrecedenceForm[HoldForm[x], 590],"
+            "PrecedenceForm[Superscript[PrecedenceForm[HoldForm[x], 590],"
```

Comment:
This is something that I guess I have corrected several times...

Reply:
@mmatera - sorry. .Then let us make sure this gets in!

Reply:
PS. Another way to make sure this gets in is to create a PR with just this change and it will go through faster.